dask/bag/text.py

```diff
@@ -119,6 +150,17 @@ def file_to_blocks(lazy_file):
             yield line
 
 
+def file_to_blocks_with_path(lazy_file):
```

Comment:
Slight preference to update `file_to_blocks` to take a boolean `include_path` and yield `(line, lazy_file.path)` if so.